### ISSUE TITLE
[Accessibility]: When clicking "Make fullscreen" button, and exiting full screen, focus is lost

### ISSUE THREAD

**Issue by amortiz** [carbon-design-system/carbon-charts#2039]
### Application/Team

IBM Cloud infrastructure

### Environment

Browser

### What happened? What did you expect to happen instead?

On any chart, when you click "make it full screen", and then you exist full screen, focus is lost, it should focus on the "make it full screen" button so users know where they are at.

Full AVT report:

```
Issue description: When the make it full screen is activated and then we came out of the full screen. Focus is lost .

Recommended fix: When the we come out of the full screen page . Focus should be on show full screen button

User impact: Users, especially those relying on keyboard navigation or assistive technologies, may find it challenging to reorient themselves on the page and continue their tasks.

WCAG Mapping: 2.4.3 Focus Order

Level of Compliance: A

Severity: Low

Test tool: JAWS_Chrome

```

<img width="980" height="551" alt="Image" src="https://github.com/user-attachments/assets/b0d80aeb-94d3-49e6-9a84-110d97b702a5" />

### What WCAG 2.1 checkpoint does the issue violate?

 2.4.3 Focus Order

### Version

Latest, just try any chart

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P3
